src/localization/successor_manager/node.h

```diff
@@ -44,11 +44,11 @@ class Node {
   void print() const;
 
   /** WARNING: use only for the priority queue.
- * Comparison is based on the accumulated cost **/
+   * Comparison is based on the accumulated cost **/
```

Comment:
This indentation looks a bit strange. Did you run `clang_format` over this file?

Reply:
Yes.